src/templates/Blog.jsx

```diff
@@ -20,6 +20,9 @@ export default class Blog extends React.Component {
       post.category_id = config.postDefaultCategoryID;
     }
     const { date } = postNode.fields;
+
+    const codeBlockRegex = /```([\s\S]*?)```/g;
+
```

Comment:
as you're not referencing `codeBlockRegex` anywhere, I assume this can be removed